dogs.html

```diff
@@ -0,0 +1,242 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href="styles/stylesheet.css">
+    <title>AdoptADog Dogs</title>
+    <meta name="author" content="Valerie V. Mehring">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  </head>
+  <body>
+    <header>
+      <h1>ADOPTADOG</h1>
+      <h2>Seattle's premiere dog adoption agency</h2>
+      <!-- <img src="images/cart-icon.svg" alt=""> -->
+      <p>$246.90</p>
+      <a href="checkout.html">
+        <button type="button" name="Checkout">Checkout</button></a>
+      <nav>
+        <a href="index.html">Home</a>
+        <a href="dogs.html">Dogs</a>
+        <a href="blog.html">Blog</a>
+      </nav>
+    </header>
+    <main>
+      <h3>Here are all of our dogs!</h3>
+      <figure>
```

Comment:
Great, i like how you laid these out. You'll need some sort of wrapper around the dogs to lay them out, but you're on the right path. I'm not 100% convinced figure is the best to use here, but that's just my opinion and you're not wrong. Honestly, I'd just wrap them in a div